ui/css/base-styles.scss

```diff
@@ -17,10 +17,7 @@ body {
 
 html {
   min-height: 500px;
-}
 
-html,
-body {
   @include design-system.screen-sm-max {
     &:not([data-theme]) {
```

Comment:
`data-theme` is applied to the `html` tag, so the `body` was inheriting these color changes regardless of whether the `data-theme` attribute existed or not.